src/renderer/components/utils/GridView.tsx

```diff
@@ -6,23 +6,32 @@
 // ==LICENSE-END==
 
 import * as React from "react";
+import { RouteComponentProps } from "react-router-dom";
 import { PublicationView } from "readium-desktop/common/views/publication";
 import * as styles from "readium-desktop/renderer/assets/styles/publicationView.css";
 import CatalogMenu from "readium-desktop/renderer/components/publication/menu/CatalogMenu";
 import OpdsMenu from "readium-desktop/renderer/components/publication/menu/OpdsMenu";
 import PublicationCard from "readium-desktop/renderer/components/publication/PublicationCard";
 
-interface IProps {
+interface Props extends RouteComponentProps {
```

Comment:
Sorry if i don't understand, but why you extends RouteComponentProps ?

Reply:
I use the location to know if it is a grid or a list display

Reply:
really sorry, i don't see other props utilisation and no hoc injection. How you used it ?

![image](https://user-images.githubusercontent.com/3198865/65889198-1dc1f100-e3a1-11e9-85c9-28123a14dfbc.png)

Reply:
Oh, sorry, I checked the wrong file...
It's true, the router props are useless here

Reply:
This comment has been addressed: https://github.com/readium/readium-desktop/pull/733/commits/cbf9576d938d2f91c9a2aef9b9f8305909f6cdc6